vars/kibanaPipeline.groovy

```diff
@@ -469,6 +469,10 @@ def allCiTasks() {
     jest: {
       workers.ci(name: 'jest', size: 'n2-standard-16', ramDisk: false) {
         scriptTask('Jest Unit Tests', 'test/scripts/test/jest_unit.sh')()
```

Comment:
I believe you need this to ensure that `runbld.junit()` is called when there are jest test failures (I believe the exception will bubble up through `scriptTask()`)

```suggestion
        catchErrors {
          scriptTask('Jest Unit Tests', 'test/scripts/test/jest_unit.sh')()
        }
```

Reply:
Can't hurt right?